docs/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = u'Scrapy'
-copyright = u'2008–2018, Scrapy developers'
+copyright = u'2008–2020, Scrapy developers'
```

Comment:
Maybe we could put the current year through code?

Reply:
It is definitely better. Done!